py3status/modules/graphite.py

```diff
@@ -66,6 +66,9 @@
     color_degraded: threshold_degraded has been exceeded
 
 @author ultrabug
+
+SAMPLE OUTPUT
+{'color': 'full_text': 'graphite'}
```

Comment:
I'm not convinced by this output.  I may try to get some real data

Reply:
yeah switch to '412 req/s' instead as graphite is meant to return numbers with units usually